### ISSUE TITLE
Status update should say how many failures

### ISSUE THREAD

**Issue by orta** [KrauseFx/danger#24]
![screen shot 2016-01-08 at 10 00 51](https://cloud.githubusercontent.com/assets/49038/12195380/b7824626-b5ee-11e5-965d-81b40ac13891.png)

What about: 
* Fails: `1 Error. 1 Warning. Everything is fixable.` (which is Danger's motto)  (don't show warning/errors if no warnings or errors )
* Success: `Everything is okay.`

I'm going to ask her about the wording now ;)

**Comment by orta** [comment]
@dangermcshane opinions on this ^

**Comment by KrauseFx** [comment]
That's btw where the text is generated https://github.com/KrauseFx/danger/blob/master/lib/danger/request_sources/github.rb#L87-L95

```ruby
    def generate_github_description(warnings: nil, errors: nil)
      if errors.count > 0
        "danger found errors"
      elsif warnings.count > 0
        "⚠️ danger found warnings, merge with caution"
      else
        "danger was successful"
      end
    end
```

**Comment by KrauseFx** [comment]
I'm okay with changing the text. I like the :warning: emoji, as it shows it's not an error, but also not a real success

**Comment by orta** [comment]
Emoji is cool :+1:

**Comment by orta** [comment]
Closed in #27 

I talked with @dangermcshane on what the status messages should say, and we came up with the results in that PR.
